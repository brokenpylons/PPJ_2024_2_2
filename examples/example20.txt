var counter = 0;
for (var i = 1, 5) {
    counter = counter + 1;
}

for (var i = 1, 3) {
    counter = counter - 1;
}

for (var i = 1, 2) {
    counter = counter + 1;
}

for (var i = 1, 4) {
    counter = counter - 1;
}

print(counter);
