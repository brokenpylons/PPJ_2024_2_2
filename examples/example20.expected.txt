variable("x") times("*") variable("y") 
